bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -160,6 +160,8 @@ pub mod pallet {
 		},
 		/// Set OperatingMode
 		OperatingModeChanged { mode: BasicOperatingMode },
+		/// A XCM message was sent.
+		Sent { destination: Location, message: Xcm<()>, message_id: XcmHash },
```

Comment:
Including the full XCM message in the event makes it too big. You can add debug logging for full XCM, but for the event `topic_id` and `message_id` should be enough.

```suggestion
		Sent { destination: Location, message_id: XcmHash, topic_id: XcmHash },
```

You can populate `topic_id` [for example like this](https://github.com/paritytech/polkadot-sdk/blob/ade1f755cbba62cd6800601d1d78202ee5f629a5/polkadot/xcm/xcm-builder/src/barriers.rs#L238).

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6965/commits/4aa1a7f976a869da41f9eab6d4209f03aecf3313